psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -264,6 +284,12 @@ def compute_hessian(self, molecule):
         #magic (if magic was easy)
         """
 
+        if self.disp_type in 'nl':
+             nat3=3*molecule.natom()
+             ZeroMat = np.zeros((nat3,nat3))
+             ZeroHess= core.Matrix.from_array(ZeroMat)
+             return ZeroHess
+
```

Comment:
Does this still have to be routed through Empirical Dispersion? I think with the dictionary functional changes this can now be skipped.

Reply:
I thought the agreement was to get NL in, and then tackle the whole Dispersion - QCDB split / rework later (perhaps for 1.3)?

Reply:
I agree for sure, just curious if this was still needed with the `dict` changes that you made. I think we might have separated these issues out.

Reply:
It can be partially avoided, i.e. still using its citation formatting powers, but avoiding the energy/gradient/hessian calls.


I do this by deleting the dispersion object `del wfn._disp_functor`, if this is acceptable.